frontend/src/component/project/Project/Project.tsx

```diff
@@ -314,6 +321,7 @@ export const Project = () => {
                 />
                 <Route path='settings/*' element={<ProjectSettings />} />
                 <Route path='metrics' element={<ProjectDoraMetrics />} />
+                <Route path='applications' element={<ProjectApplications />} />
```

Comment:
So it's available to everyone, but it's just hidden unless the flag is turned on, yeah?

Reply:
Yes